mod/community.php

```diff
@@ -92,9 +89,9 @@ function community_content(App $a, $update = 0) {
 }
 
 function community_getitems($start, $itemspage) {
-	if (get_config('system','community_page_style') == CP_GLOBAL_COMMUNITY)
+	if (Config::get('system','community_page_style') == CP_GLOBAL_COMMUNITY) {
 		return(community_getpublicitems($start, $itemspage));
```

Comment:
Standards: Can you please remove parentheses from system calls (return, echo, include)?